src/lib/utils/string.js

```diff
@@ -12,9 +12,8 @@ export const truncateMiddle = (string, maxLength = null, ellipsis = '...') => {
     return string
   }
 
-  const halfLength = Math.floor(string.length / 2 - ellipsis.length)
-  const firstHalf = string.slice(0, halfLength)
-  const lastHalf = string.slice(-halfLength)
+  const firstHalf = string.slice(0, maxLength)
+  const lastHalf = string.slice(-maxLength)
```

Comment:
```suggestion
```

Reply:
Please restore original logic. Max length should be an **maximal length of output string**
not the characters count from left/right side

if you want to  have 13 chars from left/right side, you should pass maxLength = 13 + 13 + 3 = 29.

Reply:
As we discussed with Liav and according to the design - the output string should be with a specific chars count from the left/right side.

Reply:
if you want specific  chars count from left/right side, you should pass maxLength = 2 * <chars count you need> + 3